static/js/components/tiles/histogram_tile.tsx

```diff
@@ -68,12 +69,12 @@ export function HistogramTile(props: HistogramTilePropType): JSX.Element {
   }, [props, rawData]);
 
   useEffect(() => {
-    if (histogramData) {
+    if (histogramData && !_.isEmpty(histogramData.dataPoints)) {
       draw(props, histogramData.dataPoints);
     }
   }, [props, histogramData]);
 
-  if (!histogramData) {
+  if (!histogramData || _.isEmpty(histogramData.dataPoints)) {
     return null;
   }
   const rs: ReplacementStrings = {
```

Comment:
perhaps only add the ChartTileContainer when there is data to plot (otherwise the empty frame with no chart is a little awkward).

Reply:
added this check in the new version as well.